src/services/content/content-share-handler.service.ts

```diff
@@ -86,9 +86,9 @@ export class ContentShareHandlerService {
         if (shareParams.saveFile) {
           this.commonUtilService.showToast('FILE_SAVED', '', 'green-toast');
         } else if (shareParams.byFile) {
-          let shareLink = this.appName + ': \'' + content.contentData.name + '\'' + '\n';
-          shareLink = '\n' + shareLink +
-            `\n\n${this.commonUtilService.translateMessage('TRY_CONTENT_ON')}` + '\n' + await this.getPackageNameWithUTM(true);
+          const shareLink = this.commonUtilService.translateMessage('SHARE_CONTENT_FILE', {
```

Comment:
{{app_name}}: {{content_name}} \n
Try this content on the mobile app: {{play_store_url}}